editor.planx.uk/src/@planx/components/Confirmation/Public.tsx

```diff
@@ -85,18 +74,68 @@ export default function ConfirmationComponent(props: Props) {
             <hr />
           </>
         )}
-
         {props.contactInfo && (
-          <>
-            <Box py={1}>
-              <Typography variant="h2" component="h3" gutterBottom>
-                Contact us
-              </Typography>
-              <ReactMarkdownOrHtml source={props.contactInfo} />
-            </Box>
-          </>
+          <Box py={1}>
+            <Typography variant="h2" component="h3" gutterBottom>
+              Contact us
+            </Typography>
+            <ReactMarkdownOrHtml source={props.contactInfo} openLinksOnNewTab />
+          </Box>
         )}
       </Card>
     </Box>
   );
 }
+
+function Details() {
+  const [sessionId, passport, govUkPayment, flowName] = useStore((state) => [
+    state.sessionId,
+    state.computePassport(),
+    state.govUkPayment,
+    state.flowName,
+  ]);
+
+  const details = {
+    "Application reference": sessionId,
+    "Property address": passport.data?._address?.title,
+    "Application type": [
+      flowName.replace("Apply", "Application"),
+      getWorkStatus(passport),
+    ]
+      .filter(Boolean)
+      .join(" - "),
+    "GOV.UK payment reference": govUkPayment?.payment_id,
+    "Paid at":
+      govUkPayment?.created_date &&
+      new Date(govUkPayment.created_date).toLocaleDateString("en-gb", {
+        day: "numeric",
+        month: "long",
+        year: "numeric",
+      }),
+  };
+  const applicableDetails = objectWithoutNullishValues(details) as Record<
+    string,
+    string
+  >;
+
+  return (
+    <SummaryListTable>
+      {Object.entries(applicableDetails).map(([k, v], i) => (
+        <React.Fragment key={`detail-${i}`}>
+          <Box component="dt">{k}</Box>
+          <Box component="dd">{v}</Box>
+        </React.Fragment>
+      ))}
+    </SummaryListTable>
+  );
+}
+
+// TODO - Retire in favor of ODP Schema application type descriptions or fallback to flowName
```

Comment:
Don't want to do this yet because we know our ODP Schema enums are a bit in flux, but imagine it will be the better / more consistent way forward ! This function only does special handling for LDCs - which is fine for now as they're the only live statutory service.